edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj

```diff
@@ -28,13 +28,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath=""/>
-    <None Include="images\icon.png" Pack="true" PackagePath=""/>
+    <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath="" />
+    <None Include="images\icon.png" Pack="true" PackagePath="" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.36.6" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.27" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
```

Comment:
IoT Edge 1.4 needs to stay on Microsoft.Azure.Devices.Client v1.36.x. We either need to get the SDK team to release an update to 1.36.x that bumps Newtonsoft.Json, or directly update the transitive dependency in our projects (or do the latter in the short term and the former in the long term).